docs/shipping/AWS/aws-api-gateway.md

```diff
@@ -58,7 +58,8 @@ Specify the stack details as per the table below, check the checkboxes and selec
 | `logzioListener`                           | Listener host.                                                                                                                                                                                                                                           | **Required**      |
 | `logzioType`                               | The log type you'll use with this Lambda. This can be a [built-in log type](https://docs.logz.io/docs/user-guide/data-hub/log-parsing/default-parsing/#built-in-log-types), or a custom log type.                                                                                 | `logzio_firehose` |
 | `services`                                 | A comma-seperated list of services you want to collect logs from. Supported options are: `apigateway`, `rds`, `cloudhsm`, `cloudtrail`, `codebuild`, `connect`, `elasticbeanstalk`, `ecs`, `eks`, `aws-glue`, `aws-iot`, `lambda`, `macie`, `amazon-mq`. | -                 |
-| `customLogGroups`                          | A comma-seperated list of custom log groups you want to collect logs from                                                                                                                                                                                | -                 |
+| `customLogGroups`                          | A comma-seperated list of custom log groups you want to collect logs, or the ARN of the Secret parameter (explanation below) storing the log groups list if it exceeds 4096 characters. from                                                                                                                                                                                | -                 |
```

Comment:
@8naama Secret parameter = useCustomLogGroupsFromSecret ?

Reply:
@Simplychee No, `useCustomLogGroupsFromSecret` is a parameter in the stack which indicates if they used a secret or not (`true` or `false`), if it's `true` >> then they need to place their secret ARN in `customLogGroups`
It's the explanation here:
https://github.com/logzio/documentation/pull/565/files#diff-3128cc9d1bdf5001fe5ebbcf6db9a67d65d568377fdb5f1ad07cee170b907a44R74-R86

I'll add link to it, maybe will be more clear?